tests/lib/test-plugin.h

```diff
@@ -160,13 +161,15 @@ static mptcpd_aid_t   const test_raddr_id_2    = 0x45;
 static bool           const test_backup_2      = false;
 static bool           const test_server_side_2 = true;
 static bool           const test_deny_join_id0_2 = true;
+static uint8_t        const test_error_2       = 0;
 
 static mptcpd_token_t const test_token_4       = 0x34567890;
 static mptcpd_aid_t   const test_laddr_id_4    = 0x90;
 static mptcpd_aid_t   const test_raddr_id_4    = 0x01;
 static bool           const test_backup_4      = true;
 static bool           const test_server_side_4 = false;
 static bool           const test_deny_join_id0_4 = false;
+static uint8_t        const test_error_4       = 0;
```

Comment:
small detail: maybe good to put a non 0 value for one of them, just to check something different, no? That will not change much, but just by principle, just in case?